Compile errors: no member named 'GenImageGradientV' in the global namespace

include/CPP/./Image.hpp:137:18: error: no member named 'GenImageGradientV' in the global namespace
        return ::GenImageGradientV(width, height, top, bottom);
               ~~^
include/CPP/./Image.hpp:144:18: error: no member named 'GenImageGradientH' in the global namespace
        return ::GenImageGradientH(width, height, left, right);

Compiler command:
		g++ -std=c++17 -framework CoreVideo -framework IOKit -framework Cocoa -framework GLUT -framework OpenGL libD/libraylib.a src/game.cpp -o game -Iinclude/CPP -Iinclude

--- a/include/raylib.hpp
+++ b/include/raylib.hpp
@@ -18,6 +18,10 @@ extern "C" {
 #error "raylib-cpp requires at least raylib 4.5.0"
 #endif
 
+#if RAYLIB_VERSION_MINOR > 5
+#error "raylib-cpp targets raylib 4.5. Use the `next` branch for the next version of raylib."
+#endif
+
 #ifdef __cplusplus
 }
 #endif

